src/mame/heathzenith/h19.cpp

```diff
@@ -66,7 +67,11 @@ void h19_state::h19(machine_config &config)
 ROM_START( h19 )
 ROM_END
 
+ROM_START( z19 )
+ROM_END
+
 } // anonymous namespace
 
-//    year  name     parent  compat  machine       input   class      init        company          fullname                         flags
-COMP( 1979, h19,     0,      0,      h19,          0,      h19_state, empty_init, "Heath Company", "Heathkit H-19",                 MACHINE_SUPPORTS_SAVE )
+//    year  name  parent  compat  machine input  class       init         company                fullname          flags
+COMP( 1979, h19,  0,      0,      h19,    0,     h19_state,  empty_init,  "Heath Company",       "H-19 Terminal",  MACHINE_SUPPORTS_SAVE )
+COMP( 1980, z19,  h19,    0,      h19,    0,     h19_state,  empty_init,  "Zenith Data Systems", "Z-19 Terminal",  MACHINE_SUPPORTS_SAVE )
```

Comment:
I don’t think duplicated machines are really useful if there are no differences at all.

Reply:
@rb6502 had mentioned there would be value in documenting the systems like this, but I will remove them for now so this PR can be merged (it would be trivial to add back in a later PR).


As for the copy/paste of the rom definitions, I was thinking doing something like this would be better: 
```
static const auto rom_z89 = rom_h89;
```
or have a macro in `romentry.h`
like
```
#define ROM_DUP(new_sys, existing_system)  static const auto rom##new_sys rom##existing_system;
```
So if anything changes in the rom code, it obvious that this macro needs updating.

Reply:
@rb6502 what do you think about having things like h19/z19 and h89/z89 declared even if the branding is the only difference between the systems? And a define like I mentioned above could address the copy/pasted ROM definitions Vas mentioned in the other comment.